manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -228,6 +228,11 @@ spec:
           - name: {{ $key }}
             value: "{{ $value }}"
           {{- end }}
+          {{- $network_set := index $gateway.env "ISTIO_META_NETWORK" }}
+          {{- if and (not $network_set) .Values.global.network }}
+          - name: ISTIO_META_NETWORK
+            value: {{ .Values.global.network }}
```

Comment:
nit: suggest to quote the network to avoid incorrect type for network.

Reply:
done